server/services/timelineService.ts

```diff
@@ -32,11 +16,17 @@ export default class TimelineService {
   async getTimeline(prisonNumber: string, username: string): Promise<Timeline> {
     const systemToken = await this.hmppsAuthClient.getSystemClientToken(username)
     try {
+      const timelineApiFilterOptions = new TimelineApiFilterOptions({
```

Comment:
The next set of PRs will add in the UI controls to filter by the various things, and pass those into the service, and will set the properties to true/false as necessary